staging/src/k8s.io/apiextensions-apiserver/pkg/cmd/server/server.go

```diff
@@ -48,6 +58,7 @@ func NewServerCommand(ctx context.Context, out, errOut io.Writer) *cobra.Command
 	cmd.SetContext(ctx)
 
 	fs := cmd.Flags()
+	utilversion.DefaultComponentGlobalsRegistry.AddFlags(fs)
```

Comment:
yeah, this should also go away when the composition is properly done (noted above)